supertokens/supertokens-core
# Email verification tests

## Related Issues
- 

## Changes
- Tests added:
   - EmailPassword tests:
     - Create an email verification token two times, and check that there are two entries in the db for that user with the right values
     - Verify the email successfully, then create an email verification token and check that the right error is thrown.
     - Give invalid token
     - Generate two tokens, verify with one token, the other token should throw an invalid token error
     - Use an expired token, it should throw an error
     - Test the format of the email verification token
   -  GenerateEmailVerificationTokenAPITest
     - Check for bad input (missing fields)
     - Check good input works
     - Check for all types of output 
   - VerifyEmailAPITest
     - Check for bad input (missing fields)
     - Check good input works
     - Check for all types of output
   - IsEmailVerifiedAPITest
     - Check for bad input (missing fields)
     - Check good input works
     - Check for all types of output

## Migration:
  - none
